packages/i18n/locales/en.json

```
@@ -238,9 +238,20 @@
       "tableOfContents": "Table of Contents"
     },
     "search": {
-      "searchBox": {
-        "placeholder": "Start typing..."
-      }
+      "searchPlaceholder": "Start typing...",
+      "chatPlaceholder": "Ask me anything",
+      "noResultsFoundFor": "No results found for",
+      "suggestions": "Suggestions",
+      "seeAll": "See all",
+      "addMore": "Add more",
+      "clearChat": "Clear chat",
+      "errorMessage": "An error occurred while trying to search. Please try again.",
+      "disclaimer": "Orama can make mistakes. Please verify the information.",
```

nit: this might be an opportunity to be more generic, especially coupled with `chatButtonLabel`
```suggestion
      "disclaimer": "AI summaries can make mistakes. Please verify the information.",
```

other keys here just call it a `chat service`